### PR TITLE
add:Ho Va Ten: pull 01

### PR DESCRIPTION
**Pull request by nhatnv-2504**
Pull request from Nguyen Van Nhat

### CODE REVIEW

**Comment by nhatnv-2504** [comment]
Review nhe

File: NguyenVanNhat.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Ho va ten: Nguyen Van Nhat
```
Comment (salami1995) [inline NguyenVanNhat.txt:1]:
```suggestion
Ho va ten: Nguyen Van Nhut
```

Comment (nhatnv-2504) [reply NguyenVanNhat.txt:1]: My name is Nhat, NOT Nhut

**Review by salami1995 — APPROVED** [review]
LGHF!

**Pull request merged**